### PR TITLE
add search for events

### PR DESCRIPTION
**Pull request by mykiwi**

### CODE REVIEW

File: src/AppBundle/Controller/AssetsController.php [hunk 1]
```diff
@@ -53,11 +53,14 @@ public function assetAction($path, Request $request)
      *
      * @Method("GET")
      */
-    public function mapAction(string $latitude, string $longitude)
+    public function mapAction(Request $request, string $latitude, string $longitude)
     {
         $coordinates = new Coordinates($latitude, $longitude);
 
-        if (!$contents = $this->get('app.map.google_maps_static_provider')->get($coordinates)) {
+        $contents = $request->query->has('algolia')
```
Comment (hhamon) [inline src/AppBundle/Controller/AssetsController.php]: `$contents = $this->get('app.map.google_maps_static_provider')->get($coordinates, $request->query->has('algolia') ? '250x170' : '');`

File: front/components/AlgoliaSearch.js [hunk 2]
```diff
@@ -135,9 +147,21 @@ export default class AlgoliaSearch extends React.Component {
             return `/emmanuel-macron/desintox/${hit.slug}`;
         }
 
+        if ('event' === hit.type) {
+            return `/evenements/${hit.uuid}/${hit.slug}`;
```
Comment (hhamon) [inline front/components/AlgoliaSearch.js]: backticks to enclose the string?

Comment (mykiwi) [inline front/components/AlgoliaSearch.js]: It's called [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Comment (hhamon) [inline front/components/AlgoliaSearch.js]: Oki thanks! Good to know :)

**Pull request merged**